src/neuroconv/datainterfaces/__init__.py

```diff
@@ -137,7 +142,10 @@
     SbxImagingInterface,
     TiffImagingInterface,
     Hdf5ImagingInterface,
-    ScanImageImagingInterface,
```

Comment:
I would keep the classic ScanImageImagingInterface around for both compatibility and convenience - it can be the 'best' type of autodetector duck typer we can make (has fields for both `file_path`, `folder_path` but not both -> if `file_path` given, uses single file interface and tries to detect if the file is multiplane or not, otherwise `folder_path`and again whatever best attempt we can do to guess if it's multiplane)

Reply:
Or it looks like you are

https://github.com/catalystneuro/neuroconv/pull/809/files#diff-161e12dc1cb66083907ea7e54e4fdc8f149a15b354e08d61246384d677f82131R497

without the convenience idea

so I'd still keep the import around until it's completely deprecated?

Reply:
Or perhaps simpler to just restrict the `ScanImageImagingInterface` to be `file_path` routing to single vs. multi plane

then a `ScanImageMultiFileImagingInterface` that takes `folder_path` and route to single plane or multi-plane within multi-file

then no need to deprecate and should be simpler to add than my first idea